src/lib/Preview.js

```diff
@@ -163,6 +166,13 @@ class Preview extends EventEmitter {
         this.navigateLeft = this.navigateLeft.bind(this);
         this.navigateRight = this.navigateRight.bind(this);
         this.keydownHandler = this.keydownHandler.bind(this);
+
+        this.on(PREVIEW_ERROR, (msg) => {
+            console.error(msg);
+        });
+        this.on(PREVIEW_METRIC, (msg) => {
+            console.log(msg);
```

Comment:
Do we always want to log these?

Reply:
This section is for debugging purposes only, but yes, we will always want to log PREVIEW_METRICs